homeassistant/components/honeywell/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Honeywell Total Connect Comfort (US)",
+        "description": "If you need help with the configuration please see the documentation: https://www.home-assistant.io/integrations/honeywell/.\n\nAdditional sensors (e.g. Honeywell Wireless Outdoor Sensor) will be enabled by default.",
```

Comment:
```suggestion
        "description": "Additional sensors (e.g. Honeywell Wireless Outdoor Sensor) will be enabled by default.",
```


URLs need to be placeholders here, but since the UI already has a link to the docs, it seems easier to take it out.

Reply:
After thinking about this description for a bit, I realized that it actually doesn't make any sense. Outdoor sensors aren't going to work until my next PR anyway. I think it makes more sense to make the description similar to the ecobee description.

"Please enter the credentials used to log into mytotalconnectcomfort.com."

Reply:
![image](https://user-images.githubusercontent.com/2547448/126020746-37ab7049-0cf7-4211-8464-b78ecb835ad5.png)